doc: refactor README (#54)
stacklok/codegate-ui: README.md

> # CodeGate UI Dashboard

Reviewer: ```suggestion
# CodeGate Dashboard
```

> [CodeGate](https://github.com/stacklok/codegate) UI Dashboard with some information about the security insights:

Reviewer: ```suggestion
This repository contains the [CodeGate](https://github.com/stacklok/codegate)
dashboard user interface. The dashboard presents information about CodeGate's
security insights and activity, including:

```

> - History prompts
> - Chat conversations

Reviewer: ```suggestion
- Prompt and chat conversation history
```

I think these are effectively the same thing in the UI?

Author: you're right, at the end of the day the prompts include the conversation 👍

> - Secrets alerts
> - Malicious Packages
> - Alerts trend
> - Alerts table

Reviewer: ```suggestion
- Security alert counts and daily trend
- Alert history with secrets and package risks detected by CodeGate
- CA certificate download and installation instructions
```

## Setting up local development environment

> To install all dependencies for your local development environment (endpoint on `http://localhost:8989`), run

Reviewer: ```suggestion
To install all dependencies for your local development environment (endpoint on
`http://localhost:5173`), run
```

Author: I was pointing to the API port `http://localhost:8989`, the `:5173` is the FE, I was thinking that could be important to share the API port too here. Maybe it doesn't makes sense 🤔 ?

Author: I replaced with this below, I guess sharing the env var maybe it is more clear
```
To install all dependencies for your local development environment (environment variable VITE_BASE_API_URL="http://localhost:8989" for API base url), run
```

Reviewer: Ohh I see now. I might be even more explicit, like:

```markdown
To run a local development environment, start by running a copy of the CodeGate
[application](https://github.com/stacklok/codegate/blob/main/docs/development.md)
or [container](https://docs.codegate.ai/how-to/install). The UI will connect to
the API endpoint at `http://localhost:8989` by default. To change this, set the
`VITE_BASE_API_URL` environment variable.

Install all dependencies and run a dev server:
```

Author: Right, that makes sense

```bash
npm run dev
```

> Open http://localhost:5173 on your browser to see the dashboard

Reviewer: ```suggestion
Open <http://localhost:5173> on your browser to see the dashboard
```

Reviewer: Making the bare URL explicit
